app/code/Magento/ImportExport/Model/Import/Source/FileParser/UnsupportedPathException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\ImportExport\Model\Import\Source\FileParser;
+
+class UnsupportedPathException extends \RuntimeException
+{
+    private $path;
+
+    public function __construct($path = '', $message = '')
+    {
+        parent::__construct($message ?: sprintf('Path "%s" is not supported', $path));
+
+        $this->path = $path;
+    }
+
+    public function getPath()
```

Comment:
looks like we have this public method just to use one in **Unit tests.**
We have a lot of public methods in this Pull Request which were added just to be used in Unit Tests.
They are not used in business logic.
Such approach violates the Object Encapsulation and contradicts with Object Oriented design